socorro/unittest/processor/test_breakpad_transform_rules.py

```diff
@@ -362,32 +362,29 @@ def test_external_fails_2(self, mocked_subprocess_module):
         processor_meta = get_basic_processor_meta()
 
         mocked_subprocess_handle = mocked_subprocess_module.Popen.return_value
-        mocked_subprocess_handle.stdout.read.return_value = int
+        # This data will fail in json.loads and throw an error
+        mocked_subprocess_handle.stdout.read.return_value = '{ff'
         mocked_subprocess_handle.wait.return_value = -1
 
         rule = ExternalProcessRule(config)
-
-        # the call to be tested
         rule.act(raw_crash, raw_dumps, processed_crash, processor_meta)
 
         assert processed_crash.bogus_command_result == {}
         assert processed_crash.bogus_command_return_code == -1
-        expected = [
-            'bogus_command output failed in '
-            'json: Expected String or Unicode',
-        ]
-        assert processor_meta.processor_notes == expected
+        assert (
+            'bogus_command output failed in json: Expecting property name'
+            in processor_meta.processor_notes[0]
+        )
 
 
 class TestBreakpadTransformRule2015(TestCase):
-
     def get_basic_config(self):
         config = get_basic_config()
         config.dump_field = 'upload_file_minidump'
         config.command_line = BreakpadStackwalkerRule2015.required_config.command_line.default
         config.kill_timeout = 5
         config.command_pathname = '/bin/stackwalker'
-        config.symbols_urls = 'https://localhost'
+        config.symbols_urls = ['https://example.com']
```

Comment:
This was a bug from a long while ago. `symbols_urls` needs to be a list of strings. Otherwise you end up with a command line that has `--symbol-url=h --symbolurl=t --symbolurl=t ...`